AUTHORS.md

```diff
@@ -82,3 +82,4 @@ The following people have made contributions to this project:
 - [praerien (praerien)](https://github.com/praerien)
 - [Xin Zhang (zxdawn)](https://github.com/zxdawn)
 - [Yufei Zhu (yufeizhu600)](https://github.com/yufeizhu600)
+- [Zifeng Mo (Isotr0py)](https://github.com/Isotr0py)
```

Comment:
We are sorting the names by alphabetical order of last names. So this is correct, right?